modules/home-age.nix

```
@@ -0,0 +1,135 @@
+{ config, options, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.age;
+
```

This might help with the scoping issue you noted for the `owner` option in the submodule:

```suggestion
{ config, options, lib, pkgs, ... }:

with lib;

let
  cfg = config.age;
  hmConfig = config;
```